common/src/main/java/org/apache/sedona/Functions.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.sedona;
+
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.Geometry;
+
+public class Functions {
+    public static double ST_Distance(Geometry left, Geometry right) {
+        return left.distance(right);
+    }
+
+    public static double ST_YMin(Geometry geometry) {
```

Comment:
I like the idea about applying DRY to Sedona in that scenarios,  but dont know if that naming is right in this context, maybe we should keep camel case here but only in SQL functions where we are forced we can use ST_* like naming ? WDYT ?

Reply:
I'm fine with that. Also curious about thoughts on package structure/naming